src/content/docs/browser/single-page-app-monitoring/get-started/spa-v2.mdx

```diff
@@ -0,0 +1,106 @@
+
+We're confident that Pro+SPAv2 represents a significant leap forward in SPA monitoring capabilities. By addressing past issues and embracing modern browser heuristics, we're committed to providing you with the best possible experience for monitoring your single-page applications.
+
+## Try out browser SPA monitoring 2.0 
+
+<Steps>
+
+    <Step>
+        ### Review the requirements
+
+        This feature is tested and supported inline with our standard browser support statement <link to it here>.
+
+
+    </Step>
+
+    <Step>
+        ### Add the feature flag to your browser agent
+
+        If your agent was installed with the APM method, contact our support team and we'll enable the feature flag on your account.
+        
+        If your agent was installed with NPM or the UI copy/paste method, you'll need to add a feature flag in the browser agent code:
+
+            1. Find the New Relic browser code in your webpage HTMl.
+
+            2. In the `init` configuration object, add the `soft_nav` feature flag. Here's an example:
+
+                ```js
+                <script type="text/javascript"> ;window.NREUM||(NREUM={});init={ …, feature_flags: ['soft_nav'] }:
+                ```
+
+            3. Deploy your app.
+
+        Need to disable this feature? Simply remove the feature flag.
+            
+    </Step>
+
+    <Step>
+        ### Confirm data is being sent to New Relic
+
+        The first thing to check is asking yourself “does my interaction follow the heuristic described above?” Make sure the conditions are met for the default interaction to finish.
+
+        Next, check your network tab in your browser’s dev tools. Filter for requests going to /events/1/. There should be ones for which the payload begins with bel.7;1, indicating your interaction(s) is(are) being captured and sent.
+
+        Finally, interaction data should continue to flow into your browser entity dashboards and charts.
+
+    </Step>
+
+</Steps>
+
+## Known issues [#known issues]
+
+* API usage:
+
+    * `newrelic.interaction().end()` workaround: If you previously used this to address interaction closure issues, you might now see extra, unnecessary interactions. Review your usage to ensure optimal results.
+    * `createTracer()` functionality change: `createTracer()` no longer keeps interactions open or times callbacks. If you relied on this for tracking JS code, explore alternative approaches.
+
+* AjaxRequest association:
+    * Potential exclusion from interactions: Ajax requests that initiated near the end of previous interactions might now be excluded due to shortened interaction durations.
+    * Manual extension: If you need a request to be attributed to a specific interaction, use the `waitForEnd: true` option within `newrelic.interaction()` and manually end it after the request starts.
```

Comment:
maybe re-worded to specify "to keep the interaction open until the request fires, then `.end()` the interaction. This will associate the request to the interaction." if that's more clear